src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -175,3 +175,13 @@ def reverse_resolve(self, use_case_id: UseCaseKey, org_id: int, id: int) -> Opti
         if id in REVERSE_SHARED_STRINGS:
             return REVERSE_SHARED_STRINGS[id]
         return self.indexer.reverse_resolve(use_case_id=use_case_id, org_id=org_id, id=id)
+
+    def resolve_shared_org(self, string: str) -> Optional[int]:
```

Comment:
do you want to add tests?

similar to 
https://github.com/getsentry/sentry/blob/7e768b875cea4b5764c3ac6f66ce60dd3e09bb00/tests/sentry/sentry_metrics/test_strings.py#L8-L16